README.md

```diff
@@ -89,20 +89,25 @@ EXAMPLE
 $ ./multivac/last_seen.py --branch master --branch sample-branch --format html
 ```
 
-### gather_job_data.py
+### gather_data.py
 
 SYNOPSIS
     
-    ./multivac/gather_job_data.py [OPTIONS]
+    ./multivac/gather_data.py [OPTIONS]
 
 DESCRIPTION
     
-    Get data about every completed job and group them by job ID. See the
-    parameters it collects on the
-    [website](https://www.tarantool.io/en/dev/multivac/gather_job_data/).
+    Get data about every completed job and failed test and group them by job ID.
+    See the parameters it collects on the
+    [website](https://www.tarantool.io/en/dev/multivac/gather_data/).
 
 OPTIONS
     
+    --tests, -t
+            Collect data about tests. Data collects with 
+            `multivac/sensors/test_status.py` from test logs. Without this
+            option the script will collect data only about workflows.
```

Comment:
At this level it's irrelevant which class is used to collect test data.

```suggestion
            Collect data about test failures from test logs. Without this
            option the script will collect data only about workflows.
```

Reply:
done